cql/operation/fhir-operation-cpg/src/main/java/com/ibm/fhir/operation/cpg/AbstractCqlOperation.java

```diff
@@ -137,34 +138,36 @@ protected Issue checkUnsupportedParameter(ParameterMap paramMap, String paramNam
      *            Library resource that is the entry point for the evaluation
      * @return Parameters describing the evaluation result
      */
-    protected Parameters doEvaluation(FHIRResourceHelpers resourceHelper, ParameterMap paramMap, Library primaryLibrary) {
+    protected Parameters doEvaluation(FHIRResourceHelpers resourceHelper, ParameterMap paramMap, SearchUtil searchHelper, Library primaryLibrary) {
         List<Library> libraries = LibraryHelper.loadLibraries(primaryLibrary);
-        return doEvaluation(resourceHelper, paramMap, libraries);
+        return doEvaluation(resourceHelper, paramMap, searchHelper, libraries);
     }
-    
+
     /**
      * Evaluate the requested CQL from the provided Library resource.
-     * 
+     *
      * @param resourceHelper
      *            Resource operation provider for loading related Library resources
      * @param paramMap
      *            Parameters object that describes the operation to perform
      * @param primaryLibrary
      *            Library resource that is the entry point for the evaluation
+     * @param searchHelper
+     *            A helper class for working with FHIR search parameters
      * @param libraries
      *            List of all necessary library resources. The first entry in the list is the primary library.
      * @return Parameters describing the evaluation result
      */
-    protected Parameters doEvaluation(FHIRResourceHelpers resourceHelper, ParameterMap paramMap, List<Library> libraries) {
+    protected Parameters doEvaluation(FHIRResourceHelpers resourceHelper, ParameterMap paramMap, SearchUtil searchHelper, List<Library> libraries) {

```

Comment:
I kind of want to rename SearchUtil, CompartmentUtil, *and* ParametersUtil from "util" to "helper" to reflect the change from static utilities to an instance-based class, but I held off because its already such a big changeset.  Happy to do that change if we agree everything else looks good.

Reply:
i pushed a commit with the big rename